server/bleep/src/config.rs

```diff
@@ -421,5 +421,6 @@ fn default_max_chunk_tokens() -> usize {
 }
 
 fn interactive_batch_size() -> NonZeroUsize {
-    NonZeroUsize::new(1).unwrap()
+    let batch_size = if cfg!(feature = "metal") { 5 } else { 1 };
```

Comment:
How was 5 chosen? There is some evidence that batch sizes that are powers of 2 are more efficient. Did we benchmark 4?

Reply:
5 was the smallest number that did not run out of context-memory. will test this again with 4.